variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "cluster_name" {
+ description = "REQUIRED: The name of the cluster. Should reflect the service utilizing redis."
+}
+
+variable "redis_clusters" {
+  description = "Number of Redis cache clusters (nodes) to create"
+  default = "2"
+}
+
+variable "redis_node_type" {
+  description = "Instance type to use for creating the Redis cache clusters"
+  default     = "cache.t2.small"
+}
+
+variable "redis_failover" {
+  default = "true"
+}
+
+variable "redis_version" {
+  description = "Redis version to use, defaults to 3.2.10"
+  default     = "3.2.10"
+}
+
+variable "redis_port" {
+  default = 6379
+}
+
+# variable "security_group_ids" {
+#   type        = "list"
+#   default     = []
+#   description = "A list of server side security groups"
+# }
+
+variable "apply_immediately" {
+  description = "Specifies whether any modifications are applied immediately, or during the next maintenance window. Default is false."
+  default     = "false"
+}
+
+variable "redis_maintenance_window" {
```

Comment:
Do we need to supply default values for `redis_maintenance_window` and `redis_snapshot_window` rather than let the AWS defaults get applied for us?

Reply:
Done.